docs/translations.md

```diff
@@ -0,0 +1,105 @@
+
+1. From [Crowdin](https://crowdin.com/project/houdiniproject) main page, select the language to which you want to translate.
+
+> Note: If the language still doesn't exist on Crowdin, you can request that a language gets added by [filing an issue](https://github.com/houdiniproject/houdini/issues/new?assignees=&labels=enhancement&template=language_request.md&title=%5BFEATURE%5D).
+
+2. Pick a file that you want to translate.
+
+3. Translate the strings and save.
+
+When your translations get approved, they will be automatically submitted to Houdini's repo.
+
+You can read further about Crowdin's translation editor on [their website](https://support.crowdin.com/online-editor/). You'll be notified when new strings to be translated are added to the project.
+
+## Making a component translatable
+
+Some components are still not translatable using i18n, as they are disposed in plain text on the view files.
+
+To enable translations on a page, we need to follow the steps below:
+
+1. Identify the strings that you want to make translatable.
+
+Every significant component needs to have its own translation name. For example, [app/views/roles/_new_modal.html.erb](app/views/roles/_new_modal.html.erb)'s translations were named as `roles.new`, as they refer to creating a new role.
+
+We can also have translations for strings that are shared in multiple components. For example, [config/locales/ui.en.yml](config/locales/ui.en.yml) holds UI strings that can be shared between multiple components, such as common button labels or links.
+
+2. If **there are** translation files where the strings you want to make translatable can be placed (given the context and the organization of the files), you can add the keys and original text to this file (see the example below). **After adding the new keys and strings, you can go to step 5.**
+
+Example:
+
+```yml
+en:
+  roles:
+    new:
+      invite_a_new_user: Invite a New User
+      nonprofit_admin: Nonprofit Admin
+      ...
+```
+
+3. If **there aren't** any translation files for the strings that you want to make translatable, then you need to create a new source file. The name of the translation file should be the same as the component name, but with `.en` in the and, followed by the `.yml` extension. For example, [app/views/roles/_new_modal.html.erb](app/views/roles/_new_modal.html.erb)'s translation file is named as `roles.new.en.yml`.
+
+Example:
+
+```yml
+en:
+  roles:
+    new:
+      invite_a_new_user: Invite a New User
+      nonprofit_admin: Nonprofit Admin
+      ...
+```
+
+4. Add the new file(s) you created to the [Crowdin config file](../crowdin.yml) indicating the path for the source language and the path for the translations.
+
+Example:
+
+```yml
+files:
+  - source: /**/locales/en.yml
+    translation: /**/locales/%locale%.yml
+  - source: /**/locales/roles.new.en.yml
+    translation: /**/locales/roles.new.%locale%.yml
```

Comment:
I think it would be simpler if we didn't have to add a new source file to this `crowdin.yml` file. Ultimately, the pattern we want is:
```yml
 - source: /**/locales/%part_of_file_name_before_locale%.en.yml
    translation: /**/locales/%part_of_file_name_before_locale%.%locale%.yml
```
I don't know if that's possible but it would avoid someone forgetting to add every new source file.

Reply:
There's no way to do that, sadly :(

Reply:
I think that would be possible if we reorganize the locales into directories.